src/execution/operator/filter/physical_filter.cpp

```diff
@@ -54,8 +54,8 @@ OperatorResultType PhysicalFilter::ExecuteInternal(ExecutionContext &context, Da
 
 InsertionOrderPreservingMap<string> PhysicalFilter::ParamsToString() const {
 	InsertionOrderPreservingMap<string> result;
-	result["Name"] = expression->GetName();
-	result["Estimated Cardinality"] = StringUtil::Format("%llu", estimated_cardinality);
+	result["__text__"] = expression->GetName();
```

Comment:
I get that the excessive use of headers causes a lot of noise, but the initial reason for that is for the JSONTreeRenderer, which needs a key->value. I'm afraid that `__text__` is not clear enough, can you fix up the `json_tree_renderer.cpp` as well?

`html` and `graphviz` also need a bit of work if the source `extra_text` changes like this

Reply:
I've changed it now so the tree renderer has an option `UsesRawKeyNames()` - if that is not set we convert the underscore names back into title case (so `__estimated_cardinality__` becomes `Estimated Cardinality`). The only difference for the other parsers should now be that `Stringified` has become `Text`.